packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -47,55 +103,73 @@ function carryOutConfigAction(context) {
   return result;
 }
 
-function initialize(context) {
-  return configProject(context)
-    .then(validateConfig)
-    .then((ctxt) => {
-      if (ctxt.projectConfigInfo.configValidated) {
-        return ctxt;
-      }
-      throw new Error('Invalid configuration settings');
-    });
+async function initialize(context) {
+  const { awsConfigInfo } = context.exeInfo;
+  if (context.exeInfo.inputParams[constants.Label]) {
+    const inputParams = context.exeInfo.inputParams[constants.Label];
+    Object.assign(awsConfigInfo, inputParams);
+  } else if (awsConfigInfo.configLevel === 'project' &&
+          !context.exeInfo.inputParams.yes) {
```

Comment:
Same as above. User doesn't need to explicitly mention yes flag.

Reply:
this is needed, without the yes flag it's the normal init workflow. because the cli then needs to gather information from the user.